Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: django in c:\users\andy1\appdata\local\packages\pythonsoftwarefoundation.python.3.12_qbz5n2kfra8p0\localcache\local-packages\python312\site-packages (5.2)
Requirement already satisfied: django-cors-headers in c:\users\andy1\appdata\local\packages\pythonsoftwarefoundation.python.3.12_qbz5n2kfra8p0\localcache\local-packages\python312\site-packages (4.7.0)
Requirement already satisfied: djangorestframework in c:\users\andy1\appdata\local\packages\pythonsoftwarefoundation.python.3.12_qbz5n2kfra8p0\localcache\local-packages\python312\site-packages (3.16.0)
Requirement already satisfied: asgiref>=3.8.1 in c:\users\andy1\appdata\local\packages\pythonsoftwarefoundation.python.3.12_qbz5n2kfra8p0\localcache\local-packages\python312\site-packages (from django) (3.8.1)
Requirement already satisfied: sqlparse>=0.3.1 in c:\users\andy1\appdata\local\packages\pythonsoftwarefoundation.python.3.12_qbz5n2kfra8p0\localcache\local-packages\python312\site-packages (from django) (0.5.3)
Requirement already satisfied: tzdata in c:\users\andy1\appdata\local\packages\pythonsoftwarefoundation.python.3.12_qbz5n2kfra8p0\localcache\local-packages\python312\site-packages (from django) (2025.2)
